examples/poem/redis-session/src/main.rs

```diff
@@ -20,13 +20,13 @@ async fn main() -> Result<(), std::io::Error> {
     }
     tracing_subscriber::fmt::init();
 
-    let client = Client::open("redis://127.0.0.1/").unwrap();
+    let client = Client::open("redis://0.0.0.0/").unwrap();
```

Comment:
Connecting to 0.0.0.0 isn't strictly speaking valid. (But, in practice, it should be equivalent to connecting to localhost.)